packages/auth/lib/src/widgets/auth_buttons.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:core/core.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+/// Widget for Authentication Buttons
+class AuthButtons extends StatelessWidget {
+  /// Creates widget for Authentication Buttons
+  const AuthButtons({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.stretch,
+      children: [
+        Button.secondaryOutline(
+          label: AppLocalizations.of(context)!.continueWithGoogle,
+          icon: TwitterIcons.google,
+          onPressed: () {},
+          height: 56,
+        ),
+        const SizedBox(height: 20),
```

Comment:
I think it's a bit weird that this Widget is basically three buttons in one 😅
What's the idea behind this?

Reply:
You're right it's a bit weird but it's a component in the Figma file that has 2 use cases (1 for desktop that I couldn't do now). Also later when working on responsiveness this section will move around as a whole depending on the screen size and it would be easier to have it as a separate widget.

<img width="320" alt="image" src="https://user-images.githubusercontent.com/50345358/201887771-1f43817a-0b8a-4e56-a994-a9d4cdb2eb7f.png">

Reply:
The original twitter looks a bit different 
<img width="341" alt="image" src="https://user-images.githubusercontent.com/16862002/201888118-39304ed5-c4f8-48e7-8ee7-a73d838ef0dc.png">
But in the end, we should do it as defined in the Figma, so it's fine for me